common/static/common/templates/discussion/new-post.underscore

```diff
@@ -1,5 +1,5 @@
 <form class="forum-new-post-form">
-    <ul class="post-errors" style="display: none"></ul>
+    <ul class="post-errors" style="display: none" tabindex="-1"></ul>
```

Comment:
We don't need the unordered list to be focusable, only the specific message we're sending focus to when it appears.